lib/lambda_ethereum_consensus/p2p/peerbook.ex

```diff
@@ -70,7 +70,7 @@ defmodule LambdaEthereumConsensus.P2P.Peerbook do
       |> min(len - @target_peers)
       |> max(0)
 
-    n = :rand.uniform(len)
+    n = if(len > 0, do: :rand.uniform(len), else: 0)
```

Comment:
if `len` is 0, :rand.uniform(len) crashes

Reply:
We should instead match for an empty `peerbook`, then.

```elixir
  @impl true
  def handle_info(:prune, peerbook) when map_size(peerbook) == 0 do
    schedule_pruning()
    {:noreply, peerbook}
  end

  @impl true
  def handle_info(:prune, peerbook) do
    ...
  end
```